src/force/CabanaPD_ForceModels_PMB.hpp

```diff
@@ -251,6 +251,137 @@ auto createForceModel( PMB, Fracture, ParticleType particles,
         delta, K, G0, temp, alpha, temp0 );
 }
 
+template <typename TemperatureType>
+struct ForceModel<PMB, Elastic, DynamicTemperature, TemperatureType>
+    : public ForceModel<PMB, Elastic, TemperatureDependent, TemperatureType>
+{
+    using base_type =
+        ForceModel<PMB, Elastic, TemperatureDependent, TemperatureType>;
+    using base_model = PMB;
+    using fracture_type = Elastic;
+    using thermal_type = DynamicTemperature;
+
+    using base_type::c;
+    using base_type::delta;
+    using base_type::K;
+
+    // Thermal parameters
+    using base_type::alpha;
+    using base_type::temp0;
+    double thermal_coeff;
+    double kappa;
+    double cp;
+
+    // Explicitly use the temperature-dependent stretch.
+    using base_type::thermalStretch;
+
+    // ForceModel(){};
+    ForceModel( const double _delta, const double _K,
+                const TemperatureType _temp, const double _kappa,
+                const double _cp, const double _alpha,
+                const double _temp0 = 0.0 )
+        : base_type( _delta, _K, _temp, _alpha, _temp0 )
+    {
+        set_param( _delta, _K, _kappa, _cp, _alpha, _temp0 );
```

Comment:
Would it be better to have "_kappa, _cp" after the mechanics related parameters at the end?

Reply:
Does the order matter?

Reply:
Not really. Actually, perhaps this order is better, since we have:
delta: nonlocal parameter
K: "pure" mechanics parameter
kappa, c_p: "pure" thermal parameters
alpha, temp0: thermo-mechanics coupling parameters